scss/tokens/_raw.scss

```
@@ -1,82 +1,407 @@
 @import "../color-palette";
 
-//// Raw primitive values
-//// Insertion of brand foundations
+// Raw primitive values
+// Insertion of brand foundations
 
 // Border
 
 // scss-docs-start ouds-raw-border
-$ouds-border-base:        4px !default;
-
-$ouds-border-radius-0:    0 !default;
-// $ouds-border-radius-25:   $ouds-border-base * .25 !default;
-// $ouds-border-radius-50:   $ouds-border-base * .5 !default;
-$ouds-border-radius-75:   $ouds-border-base * .75 !default;
-// $ouds-border-radius-100:  $ouds-border-base !default;
-$ouds-border-radius-150:  $ouds-border-base * 1.5 !default;
-// $ouds-border-radius-200:  $ouds-border-base * 2 !default;
-$ouds-border-radius-300:  $ouds-border-base * 3 !default;
-// $ouds-border-radius-400:  $ouds-border-base * 4 !default;
-// $ouds-border-radius-500:  $ouds-border-base * 5 !default;
-// $ouds-border-radius-600:  $ouds-border-base * 6 !default;
-// $ouds-border-radius-800:  $ouds-border-base * 8 !default;
+$ouds-border-base: 4px !default;
 $ouds-border-radius-9999: 2000px !default;
+$ouds-border-style-dashed: "dashed" !default;
+// $ouds-border-style-dotted: "dotted" !default;
+// $ouds-border-style-none: "none" !default;
+$ouds-border-style-solid: "solid" !default;
+$ouds-border-radius-0: $ouds-border-base * 0 !default; // 0
```

```suggestion
$ouds-border-radius-0: strip-units($ouds-border-base * 0) !default; // 0
```
Is there any way to do it on Tokenator side ? I mean when it's `* 0` ?